oci/oci-how-to/deploy-pro-container-on-pro-kubernetes-cluster.rst

```diff
@@ -0,0 +1,202 @@
+
+	.. tab:: Nutanix
+
+		See the `Nutanix Karbon Guide <https://portal.nutanix.com/page/documents/details?targetId=Karbon-v2_4:kar-karbon-deploy-cluster-c.html>`_
+		to learn how to create a cluster.
+
+
+		**Limitations**
+
+		Nutanix does not have Ubuntu Pro offerings for the nodes, i.e. you cannot
+		choose Ubuntu Pro images for the nodes.
+
+			"Deploying Kubernetes clusters in Karbon requires a **CentOS** image.
+			You must choose from a CentOS version and download the image."
+
+			-- `Nutanix docs: "Downloading Images" <https://portal.nutanix.com/page/documents/details?targetId=Karbon-v2_4:kar-karbon-upload-image-t.html>`_
+
+
+
+		**Get an Ubuntu Pro subscription**
+
+		You can reach out `here <https://ubuntu.com/support/contact-us?product=contextual-footer-ua>`_
+		to attach the cluster nodes to an Ubuntu Pro subscription, or contact
+		`rocks@canonical.com <mailto:rocks@canonical.com>`_ if you need additional support.
+
+
+		**Success**
+
+		After attaching your Ubuntu Pro subscription to the cluster nodes, you will have an
+		Ubuntu Pro Kubernetes cluster running on Nutanix.
+
+
+Create a Secret for Private Registry
+------------------------------------
+
+If you published your image to a private registry, you will need to create a
+`secret <https://kubernetes.io/docs/concepts/configuration/secret/>`_ in Kubernetes.
+`See here <https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/>`_
+to learn about pulling images from private registries.
+
+You can use the following example command to create a secret named ``regcred`` for
+`Docker Hub <https://hub.docker.com>`_. For this, use ``https://index.docker.io/v1/``
+as ``<your-registry-server>``.
+
+::
+
+	kubectl create secret docker-registry regcred \
+	    --docker-server=<your-registry-server> \
+	    --docker-username=<your-name> \
+	    --docker-password=<your-pword> \
+	    --docker-email=<your-email>
+
+
+Deploy Pro container image to Pro Kubernetes cluster
+----------------------------------------------------
+
+You can deploy your Pro container image in a `Pod`_, `Deployment`_, or as a `Service`_.
+Make sure to include your created `secret`_ and your Pro container image correctly.
+
+.. _Pod: https://kubernetes.io/docs/concepts/workloads/pods/
+.. _Deployment: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/
+.. _Service: https://kubernetes.io/docs/concepts/services-networking/service/
+
+Here is a manifest for an example Pod that consists of a container running your
+Ubuntu Pro image. It uses your secret ``regcred`` to pull the Pro container image
+from your private registry.
+
+::
+
+	# pro-container-pod.yaml
+	apiVersion: v1
+	kind: Pod
+	metadata:
+	  name: pro-container-pod
+	spec:
+	  imagePullSecrets:
+	  - name: regcred
+	  containers:
+	  - name: ubuntu-pro-container
+	    image: <your-private-pro-image>
+	  restartPolicy: OnFailure
+
+Replace ``<your-private-pro-image>`` with the path to an image in a private registry
```

Comment:
```suggestion
Replace ``<your-private-pro-image>`` with your private Pro container image (something
similar to ``your.private.registry.example.com/janedoe/jdoe-private:v1``).
```

Reply:
Please use a more realistic example. The secret created above is meant to work with Docker Hub, so provide a mock example of a private image that would be hosted there.

Reply:
i have put a realistic example